compatibility_server/pip_checker.py

```diff
@@ -50,6 +57,34 @@
     r'(.*)has requirement(.*)but you have(.*)')
 
 
+PROJECT_ID = 'python-compatibility-tools'
```

Comment:
You'll have to calculate or otherwise parameterize this somehow.

Reply:
done